src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -1,279 +1,400 @@
+import apiClient from "../Auth/apiClient";
+import AggregationTable from "./AggregationTable";
+import { v4 as uuidv4 } from "uuid";
+import { UserSchema } from "../../schemas/UserSchema";
 
-//TODO - Refactor to backend calls once setup to pull rows, etc. 
+import { SearchIcon, WarningIcon, DownloadIcon } from "@chakra-ui/icons";
+import { Select, components } from "chakra-react-select";
 
+//TODO - Refactor to backend calls once setup to pull rows, etc.
 
-//To test uploading a timesheet 
-// apiClient.updateUserTimesheet(testingTimesheetResp); 
+//To test uploading a timesheet
+// apiClient.updateUserTimesheet(testingTimesheetResp);
 
 const createEmptyTable = (startDate, company) => {
-    // We assign uuid to provide a unique key identifier to each row for reacts rendering 
-    
-    //TODO's: Pull UserID automatically
-    return {
-        UserID: "77566d69-3b61-452a-afe8-73dcda96f876",
-        TimesheetID: uuidv4(), 
-        CompanyID: company, 
-        StartDate: startDate, 
-        Status: {
-            Stage: Review_Stages.APPROVED, 
-            Timestamp: undefined 
-        },
-        WeekComments: [], 
-        TableData:[], 
-        ScheduledData: undefined 
-    }
-}
-
-const user = 'Example User'
+  // We assign uuid to provide a unique key identifier to each row for reacts rendering
+
+  //TODO's: Pull UserID automatically
+  return {
+    UserID: "77566d69-3b61-452a-afe8-73dcda96f876",
+    TimesheetID: uuidv4(),
+    CompanyID: company,
+    StartDate: startDate,
+    Status: {
+      Stage: Review_Stages.APPROVED,
+      Timestamp: undefined,
+    },
+    WeekComments: [],
+    TableData: [],
+    ScheduledData: undefined,
+  };
+};
+
+const user = "Example User";
 
 const testingEmployees = [
-    {UserID: "abc", FirstName: "joe", LastName: "jane", Type: "Employee", Picture: "https://www.google.com/koala.png"},
-    {UserID: "bcd", FirstName: "david", LastName: "lev", Type: "Employee", Picture: "https://www.google.com/panda.png"},
-    {UserID: "cde", FirstName: "crys", LastName: "tal", Type: "Employee", Picture: "https://www.google.com/capybara.png"},
-    {UserID: "def", FirstName: "ken", LastName: "ney", Type: "Employee", Picture: "https://www.google.com/koala.png"},
-]
-
-function ProfileCard({employee}) {
-
+  {
+    UserID: "abc",
+    FirstName: "joe",
+    LastName: "jane",
+    Type: "Employee",
+    Picture: "https://www.google.com/koala.png",
+  },
+  {
+    UserID: "bcd",
+    FirstName: "david",
+    LastName: "lev",
+    Type: "Employee",
+    Picture: "https://www.google.com/panda.png",
+  },
+  {
+    UserID: "cde",
+    FirstName: "crys",
+    LastName: "tal",
+    Type: "Employee",
+    Picture: "https://www.google.com/capybara.png",
+  },
+  {
+    UserID: "def",
+    FirstName: "ken",
+    LastName: "ney",
+    Type: "Employee",
+    Picture: "https://www.google.com/koala.png",
+  },
+];
+
+
+// size - s,m,l
+function AuthorComponent({ user , size }) {
```

Comment:
I'd prefer to put this in a separate file so that it's easier to find and this TimeSheet.tsx file is easier to read. You'd also have the space to make an interface for these props.